code/modules/vehicles/armored/_multitile.dm

```diff
@@ -28,6 +28,40 @@
 	)
 	///pass_flags given to desants, in addition to the vehicle's pass_flags
 	var/desant_pass_flags = PASS_FIRE|PASS_LOW_STRUCTURE
+	///particle holder for smoke effects
+	var/obj/effect/abstract/particle_holder/smoke_holder
+	///Holder for smoke del timer
+	var/smoke_del_timer
+
+/obj/vehicle/sealed/armored/multitile/Destroy()
+	QDEL_NULL(smoke_holder)
+	return ..()
+
+/obj/vehicle/sealed/armored/multitile/update_name(updates)
+	. = ..()
+	name = initial(name)
+	if(armored_flags & ARMORED_IS_WRECK)
+		name = "wrecked " + name
+
+/obj/vehicle/sealed/armored/multitile/update_desc(updates)
+	. = ..()
+	desc = initial(desc)
+	if(armored_flags & ARMORED_IS_WRECK)
+		desc += " Now just a smouldering ruin."
+
```

Comment:
Correct me if I'm wrong but shouldn't this be editing the return

Reply:
Name too

Reply:
Nah, these actually just direct set it. Parent call is just for the sig